app/src/main/res/drawable/color_cursor.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-  <size android:width="0.5dp" />
+  <size android:width="2dp" />
```

Comment:
> Why was this changed? And also how does this affect the light-mode?

Since in develop branch the size for cursor is 2dp which is not handled by this xml file so to keep it consistent I changed the width from 0.5 to 2 dp.

| 0.5dp (current branch) | 2dp (develop branch) |
|-------|---|
|<img src="https://user-images.githubusercontent.com/78796264/178481561-ba637737-76fe-4bd9-8abe-e4a593d56d3d.jpg" width="280" />|<img src="https://user-images.githubusercontent.com/78796264/178481428-336f124c-ef9a-4c15-9e56-d0ab81c562dd.jpg" width="280" />|

Reply:
Great. Thanks.